include/dlaf/communication/error.h

```diff
@@ -11,36 +11,36 @@
 #pragma once
 
 /// @file
+//
+#include "dlaf/common/utils.h"
+
+#include <mpi.h>
 
-#include <cstdio>
 #include <iostream>
 #include <memory>
 
-#include <mpi.h>
-
 namespace dlaf {
 namespace internal {
 
 /// MPI provides two predefined error handlers
 ///
-/// - `MPI_ERRORS_ARE_FATAL` (the default ?): causes MPI to abort
+/// - `MPI_ERRORS_ARE_FATAL` (the default): causes MPI to abort
 /// - `MPI_ERRORS_RETURN`: causes MPI to return an error values instead of aborting
 ///
 /// This function is only relevant when the error handler is `MPI_ERRORS_RETURN`. To set the error
 /// handler use `MPI_Errhandler_set`.
 ///
-inline void mpi_call(int err, char const* file, int line) {
+inline void mpi_call(int err, common::internal::source_location const& info) {
```

Comment:
I see it was already east-const, so it is not your choice. I'd like to know your opinion, @teonnik @rasolca about adding (or not) a rule to the coding style document for trying to keep uniform the codebase.

Reply:
The code base is mainly using west-const. (as all the example in the coding style even if it is not explicit.)
@albestro I think the coding style needs a full review.

Reply:
Regarding east-west const, I will use west-const as it is the style used in the rest of the code. There is a clang-format option in the works : https://reviews.llvm.org/D69764